### ISSUE TITLE
Additions required

### ISSUE THREAD

**Issue by rohit-mp** [rohit-mp/linux#2]
### Description
This repository was created as a means to help first-time linux users or people with little experience in using linux/unix systems get adjusted to the command line by learning new commands to complete the set of tasks given.

### Task
Create new task for the users to solve by introducing them to new commands or a combination of the known commands ( which they would have previously learnt by solving the already given tasks ).
Also add the solutions to the new tasks added in the solutions.md file in the root directory of this repo.

### Details
- Requirements : Knowledge of terminal commands, bash script(optional)
- Type of Issue : Multiple
- Time limit : 4 days

#### Note
- Claim this issue in the comments below before starting work on it
- If you will be modifying any of the existing tasks to improve upon them or to create better new tasks, make necessary changes in the solutions.md file too.
- The issue will be reassigned if the issue hasn't been solved within 4 days.
- Keep in mind that this is being done to help first-time linux users to get used to the command terminal.
